src/BodyPipe.cc

```diff
@@ -448,8 +448,6 @@ const char *BodyPipe::status() const
     static MemBuf outputBuffer;
     outputBuffer.reset();
 
-    outputBuffer.append(" [", 2);
```

Comment:
BodyPipe is not an AsyncJob. It should not be modified in a PR focusing on mgr:jobs. Please check for other similar cases.